waku/node/peer_manager/peer_manager.nim

```diff
@@ -412,6 +412,10 @@ proc dialPeer(
   trace "Dialing peer", wireAddr = addrs, peerId = peerId, proto = proto
 
   # Dial Peer
+  if proto == "":
+    await pm.switch.connect(peerId, addrs)
+    return none(Connection)
```

Comment:
I think it is better to create a new proc `connectPeer` in this module.
It is a bit misleading having a `dialPeer` proc which is actually performing a `switch.connect` ;P

Also, I like the idea of having a mandatory `proto` argument for `dialPeer`.